packages/teams-js/test/public/people.spec.ts

```diff
@@ -45,87 +49,96 @@ describe('people', () => {
     it('should not allow selectPeople calls before initialization', () => {
       expect(() => people.selectPeople()).toThrowError('The library has not yet been initialized');
     });
+
     Object.values(FrameContexts).forEach(context => {
       if (allowedContexts.some(allowedContext => allowedContext === context)) {
-        it(`should allow selectPeople calls with null peoplePickerInputs. context: ${context}`, async () => {
-          await mobilePlatformMock.initializeWithContext(context);
-          mobilePlatformMock.setClientSupportedSDKVersion(minVersionForSelectPeople);
-          people.selectPeople(null);
-          const selectPeopleMessage = mobilePlatformMock.findMessageByFunc('people.selectPeople');
-          expect(selectPeopleMessage).not.toBeNull();
-          expect(selectPeopleMessage.args[0]).toEqual(null);
-        });
+        Object.values(v1HostClientTypes).forEach(hostClientType => {
```

Comment:
Oh, did we decide to run the tests against all the hostCleintTypes (instead of framed/framless)? 

v1HostClientType contains desktop which can be frameless or framed, but we are always initializing it with framelessPlatformMock.

Reply:
For People capability we have support in specific host client type and during unit testing we don't get runtime configuration. So, the work around is to provide host client type while initialization.

Reply:
I think a comment in the tests would be super helpful!

Reply:
Right now, for web, we are initializing with frameless and passing web as a hostClientType when we know web is framed. 
Does it make sense to initialize using utils for web? and similar changes for others?

Reply:
I don't want to create unnecessary work for you. may be @erinha can suggest/comment?

Reply:
For this PR we can keep it to isSupported work and later we can create another task for it.

Reply:
That is a good idea :)